### ISSUE TITLE
Can't use backspace in editmode on a Mac

### ISSUE THREAD

**Issue by svenpopping** [goalhub/simpleIDE#2]
It isn't possible to use the backspace key when you're editing a file in the simpleIDE on a Mac OS X machine.

```
MacBook Pro (Retina, 13-inch, Mid 2014)
OS X Yosemite 10.10.4
```

**Comment by Wouter1** [comment]
Yes. It's a known issue #3439. Somehow the editor fails to load the syntax definition file  (you can see the stacktrace in the console output) and that causes the editor to misbehave.

**Comment by Venorcis** [comment]
#3439 refers to a ticket on Trac (http://ii.tudelft.nl/trac/goal/ticket/3439)

This issue is on every OS (not Mac-specific)
